BaseListener/Helpers/GeneralModels/PaginatedResponse.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using BaseListener.Boundary.Response;
+
+namespace BaseListener.Helpers.GeneralModels
+{
+    public class PaginatedResponse<T>
+    {
+        [JsonPropertyName("results")]
+        public IEnumerable<T> Results { get; set; }
+
+        [JsonPropertyName("metadata")]
+        public MetadataModel Metadata { get; set; }
+    }
+}
```

Comment:
I forgot to comment on the previous paging models.

Maybe we should import them from our finance nuget package instead?

Reply:
The name of the package is:
https://github.com/LBHackney-IT/finance-shared

Here it's Nuget page:
https://github.com/LBHackney-IT/finance-shared/pkgs/nuget/Hackney.Shared.Finance

Once you add though, you'll need to set up the LBHPACKAGESTOKEN like you did for transactions api.

The package should contain all the models you need: **PaginatedResponse<T>, MetadataModel, PagingDetails** _(used to be called **Pagination**, but had to be renamed due to name clash with the namespace)_ These models can be under different namespaces within the package, so feel free to look through the package repo to see what their namespaces are.

Reply:
Done